client/public/about.html

```diff
@@ -0,0 +1,261 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="e-ial-scale=1" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Righteous&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="stylesheet" type="text/css" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
+    />
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"></script>
+    <title>DanceWithMe</title>
+  </head>
+  <body>
+    <nav class="navbar navbar-expand-sm bg-light navbar-light">
+      <!-- Brand/logo -->
+      <a class="navbar-brand" href="http://localhost:3000/">
+        <img
+          src="dancing.png"
+          alt="logo"
+          style="width: 30px; height: 30px;"
+        />DanceWithMe
+      </a>
+
+      <!-- Links -->
+      <ul class="navbar-nav">
+        <li class="nav-item">
+          <a class="nav-link" href="http://localhost:3000/">Home</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="">About</a>
+        </li>
+      </ul>
+    </nav>
+    <div class="bg-light">
+      <div class="container py-5">
+        <div class="row h-100 align-items-center py-5">
+          <div class="col-lg-6">
+            <h1 class="display-4">Who Are We?</h1>
+            <p class="lead text-muted mb-0">
+              We are a team from Class #6.
+            </p>
+            <p class="lead text-muted">
+              in
+              <a href="https://hackyourfuture.be/" class="text-muted">
+                <u>Hack Your Future Belgium</u></a
+              >
+            </p>
+          </div>
+          <div class="col-lg-6 d-none d-lg-block">
+            <img src="hyf.jpeg" alt="" class="img-fluid rounded" />
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="bg-white py-5">
+      <div class="container py-5">
+        <div class="row align-items-center mb-5">
+          <div class="col-lg-6 order-2 order-lg-1">
+            <i class="fa fa-bar-chart fa-2x mb-3 text-primary"></i>
+            <h2 class="font-weight-light">Our Goal</h2>
+            <p class="font-italic text-muted mb-4">
```

Comment:
We created an alternative platform for dance lovers who would like to have partners to enjoy the amazing dance events with. There are applications such as "meetup.com" or "eventbrite.com" where you can see hundreds of events. If you do not have a partner, it makes no sense to join those dance events. Also there are various matching platforms such as sportpartner.com where you can search for partners. But it makes no sense to find a dance partner if there is no dance event you can go with. Besides, it is time consuming to search for ideal partners in those websites.

Reply:
@u-uysal instead of the paragraph below, I recommend this paragraph